Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -1,6 +1,16 @@
 module Hardware
   class CPU
     class << self
+      OPTIMIZATION_FLAGS_LINUX = {
```

Comment:
> LINUX_OPTIMIZATION_FLAGS would be more consistent with PPC_OPTIMIZATION_FLAGS.

:+1:

> I wonder if it's worth just overriding the dunno in OPTIMIZATION_FLAGS instead? The other values look good?

I've updated the PR to get the values for `LINUX_OPTIMIZATION_FLAGS` from `OPTIMIZATION_FLAGS`.